src/main/java/core/basesyntax/service/report/ReportServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.report;
+
+import core.basesyntax.dao.Storage;
+import core.basesyntax.model.FruitRecord;
+import java.util.List;
+
+public class ReportServiceImpl implements ReportService {
+    private static final String FRUIT = "fruit";
+    private static final String QUANTITY = "quantity";
+    private static final String COMMA = ",";
+
+    @Override
+    public String getReport(List<FruitRecord> fruitRecords) {
+        StringBuilder report = new StringBuilder();
+        report.append(FRUIT).append(COMMA).append(QUANTITY).append(System.lineSeparator());
+
+        Storage.storage.forEach((fruit, quantity) -> report
```

Comment:
The use of `forEach` here is not encapsulating the logic of how the report is generated. Consider abstracting this logic into a method that formats a single line, and then use that method within the loop.